### PR TITLE
Add neutral review. And some more dummy comments.

### PR DESCRIPTION
**Pull request by mgsergio**
@ygorshenin @milchakov

### CODE REVIEW

File: map/booking_api.hpp [hunk 1]
```diff
@@ -27,12 +27,12 @@ class BookingApi
     HotelReview() = default;
     // C++11 doesn't allow aggragate initialization for structs with default member initializer.
     // But C++14 does.
-    HotelReview(string const & reviewPositive,
-                string const & reviewNegative,
-                string const & author,
-                string const & authorPictUrl,
-                float const rating,
-                time_point<system_clock> const date)
+    explicit HotelReview(string const & reviewPositive,
```
Comment (ygorshenin) [inline map/booking_api.hpp]: No need in explicit here.

File: map/booking_api.hpp [hunk 2]
```diff
@@ -42,11 +42,26 @@ class BookingApi
     {
     }
 
+    explicit HotelReview(string const & reviewNeutral,
```
Comment (ygorshenin) [inline map/booking_api.hpp]: Same here.

**Comment by mgsergio** [comment]
PTAL

File: map/booking_api.hpp (hunk 1 above)
Comment (mgsergio) [inline map/booking_api.hpp]: Done.

File: map/booking_api.hpp (hunk 2 above)
Comment (mgsergio) [inline map/booking_api.hpp]: Done.

**Comment by milchakov** [comment]
LGTM

File: map/booking_api.hpp [hunk 3]
```diff
@@ -29,24 +29,64 @@ class BookingApi
...
+      , m_reviewNeutral(reviewNeutral)
       , m_author(author)
       , m_authorPictUrl(authorPictUrl)
       , m_rating(rating)
       , m_date(date)
     {
     }
 
+
+    static HotelReview CriticReview(string const & reviewPositive,
```
Comment (ygorshenin) [inline map/booking_api.hpp]: s/Critic/PositiveNegative/
?

Comment (mgsergio) [inline map/booking_api.hpp]: I've considered this one and I did't like it.

**Comment by ygorshenin** [comment]
LGTM

File: map/booking_api.hpp (hunk 3 above)
Comment (ygorshenin) [inline map/booking_api.hpp]: Okay, let's leave this as is.

**Pull request merged**